writer_test.go

```diff
@@ -179,15 +179,15 @@ func TestWriteWithProvidedPriority(t *testing.T) {
 	defer w.Close()
 
 	var bytes int
-	bytes, err = w.WriteWithPriority(LOG_DEBUG, []byte("this is a test message"))
+	bytes, err = w.WriteWithPriority(LOG_DEBUG|LOG_LOCAL5, []byte("this is a test message"))
```

Comment:
I'd like to add a new test rather than changing this one.

(IE, leave the test that shows `LOG_DEBUG` continuing to work as it did before so we verify that we're not regressing, with a new test showing `LOG_DEBUG|LOG_LOCAL5` working to verify the new/improved functionality.)